docs/_steps/saving_workflow_step.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Saving the step configuration
+lang: en
+slug: saving-steps
+order: 4
+---
+
+<div class='section-content'>
+
+When the workflow step's configuration has been saved (using the step configuration modal from the `edit` callback), your app will listen for the `view_submission` event. The method assigned to the `save` property of the `WorkflowStep` configuration object passed in during instantiation will run when this event occurs.
```

Comment:
Hmm this first sentence confuses me a bit since the section is "saving steps" and it starts with "when it has been saved"

Maybe "After the configuration modal is opened, your app will listen for the `view_submission` event" or something along those lines?